### ISSUE TITLE
Check Style and Analyse Wrench scripts - reduce the hud timeout

### ISSUE THREAD

**Issue by Phuket2** [omz/Pythonista-Issues#353]
It seems to me that the hud is on the screen about twice as long as it should be for 'Check Style' and 'Analyse' wrench functions, on success.  Small thing, but can get annoying. So this should be added last on the todo list :) still its just worth mentioning.

**Comment by cclauss** [comment]
That delay is designed into Pythonista so that you can take a sip of your favorite beverage as a reward for producing such clean code.

**Comment by Phuket2** [comment]
@cclauss , how do you sleep at night :) but @cclauss this is your sh*t storm.....I would not have noticed if you haven't been so hard on me. 
Joking aside, its working out very nicely.  But it is taking time.  Many things going on at the moment.

**Comment by zrzka** [comment]
Agree, would be nice if the appearance will be little bit shorter.
